fbw-a32nx/src/systems/fmgc/src/navigation/Navigation.ts

```diff
@@ -66,7 +68,8 @@ export class Navigation implements NavigationProvider {
 
   currentPerformance: number | undefined;
 
-  accuracyHigh: boolean = false;
+  private readonly _accuracyHigh = Subject.create(false);
+  public readonly accuracyHigh = this._accuracyHigh as Subscribable<boolean>;
```

Comment:
I think we can avoid the typecast like this
```suggestion
  public readonly accuracyHigh: Subscribable<boolean> = this._accuracyHigh;
```

Reply:
Never considered that as we always do the cast at work... but I can't see any obvious problem with it.. 👍